examples/src/bin/rpk_example.rs

```diff
@@ -0,0 +1,390 @@
+use log::trace;
```

Comment:
Can you add a `//!` comment at the top of this file that roughly matches the way the other examples introduce themselves?

I think `examples/README.md` could also use an update.